api/v1_stats.inc

```diff
@@ -11,10 +11,22 @@ function api_v1_stats_site($method, $data, $query_params)
     $res = DPDatabase::query("SELECT COUNT(*) FROM users");
     list($registered_users) = mysqli_fetch_row($res);
 
-    return [
+    $stats = [
         "server_time" => date(DATE_ATOM),
         "registered_users" => $registered_users,
     ];
+
+    foreach ( array(1,7,30) as $days_back )
+    {
+        $res = DPDatabase::query("
+            SELECT COUNT(*)
+            FROM users
+            WHERE t_last_activity > UNIX_TIMESTAMP() - $days_back * 24*60*60
+        ");
+        list($num_users) = mysqli_fetch_row($res);
+        $stats[$days_back . "_day_active_users"] = $num_users;
```

Comment:
I would probably swap the order of the key so that it doesn't start with a number such that it would be a valid variable name should some consumer want to do so: `active_users_1_day`, etc.

Reply:
The tyranny of C-style identifiers :-)